fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/StuckJobExecutorServiceImpl.java

```diff
@@ -111,4 +133,57 @@ private boolean areAllPartitionsCompleted(Long stuckJobId, String partitionerSte
         Long notCompletedPartitions = jobExecutionRepository.getNotCompletedPartitionsCount(stuckJobId, partitionerStepName);
         return notCompletedPartitions == 0L;
     }
+
+    private void markAsCompleted(long executionId) {
```

Comment:
We cannot mark it as COMPLETED... it was not COMPLETES. We shall mark it as FAILED!

Reply:
Done, Status will be Failed